.github/workflows/ci.yml

```diff
@@ -15,36 +15,16 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
 
-  fetch_test_deps:
-    runs-on: ubuntu-20.04
-
-    # Steps represent a sequence of tasks that will be executed as part of the job
-    steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
-
-      - name: Mount bazel repository cache
-        uses: actions/cache@v3
-        with:
-          path: "~/bazel/repository_cache"
-          key: ${{ runner.os }}-bazel
-
-      - name: Fetch external test deps
-        run: |
-          bazelisk --bazelrc=ci.bazelrc fetch @com_google_googletest//... \
-            @com_github_google_benchmark//... \
-            @com_google_absl//...
-
```

Comment:
I'm not sure that removing this is a great idea. I'm going to re-run the CI a couple of times to see how well it does with caching. I recently refactored this to ensure that I could build the compilation tests in parallel github actions, while sharing the repository and compilation caches. If you have a cache for each seperate entry in the github matrix it quickly reach's the upper limit of 10GB, and the cache's get evicted.

it might be doable to remove this section if the cache keys are configured differently https://github.com/bazelembedded/rules_cc_toolchain/pull/45/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR44, but I'm not sure.

Getting caching right can be the difference between a 5-10min CI run compared to a 1-2hr run.

Reply:
I was wrong. This caches totally fine, and the caches build times are about 5min or so which is great!